test/domains/vass/domain-delete.chpl

```diff
@@ -0,0 +1,4 @@
+// formerly BenA's types/records/compilerErrors/deleteRecordMessage
```

Comment:
The test invokes `delete` on a domain, right? I do not see what that has to do with records.